x-hooks/use-api.tsx

```diff
@@ -72,7 +73,7 @@ export default function useApi() {
     repoId = "",
     time = "",
     state = "",
-    sortBy = "updatedAt",
+    sortBy = "createdAt",
```

Comment:
should we change the default, or should we just use `createdAt` as a param when calling this function to that case?